docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -203,7 +210,7 @@ After selecting the image, you will see all of the data that is associated with
 
 [Add tags](/manage/data/label/#image-tags) for each of the puzzle pieces.
 Type in your desired tag in the **Tags** section and save the tag.
-Since Tess wanted to classify their toys by shape, they used “octagon”, “circle”, “triangle”, “oval”, “rectangle”, “pentagon”, “diamond”, and “square”.
+Since Tess wanted to classify their toys by shape, they used “octagon,” “circle,” “triangle,” “oval,” “rectangle,” “pentagon,” “diamond,” and “square”.
```

Comment:
why did you change this? the comma should not be in the quote.

Reply:
Oh I made the suggestion to change it to comma inside quote because since these are not inside code backticks, I figured they'd follow normal punctuation rules.

Reply:
Comma within quote is indeed a common copy rule, but I see it most frequently when quoting people verbatim, such that a single pair of quotes is used to quote a person's statement (incl their chosen punctuation). Here, these are values representing classification terms, and the quotation marks I think are in line with how they appear in JSON: quoted strings. I think commas within then makes for a confusing read: we are not quoting 8 person quotes, but 8 value strings. In code context, the commas would indeed be outside the quoted JSON values.

Alternatively, I think these could just be monospace if simpler. They represent strings I've entered verbatim into the text area on this App page, and monospace helps me to know that the exact match is important, and that these are not, say, placeholders or example suggestions (they all map to the chosen model).